packages/host/app/components/operator-mode/card-inheritance-panel.gts

```diff
@@ -28,8 +30,20 @@ interface Args {
 }
 
 export default class CardInheritancePanel extends Component<Args> {
-  @tracked cardInstance: CardDef | undefined;
-  @service declare operatorModeStateService: OperatorModeStateService;
+  @service private declare operatorModeStateService: OperatorModeStateService;
+  @tracked private lastModified: string | undefined;
+  private refreshSaveMsg: number | undefined;
+
+  constructor(owner: unknown, args: any) {
+    super(owner, args);
+    this.refreshSaveMsg = setInterval(
+      () => this.calculateLastModified(),
+      10 * 1000,
+    ) as unknown as number;
+    registerDestructor(this, () => {
+      clearInterval(this.refreshSaveMsg);
```

Comment:
Probably not important but does `refreshSaveMsg` even need to be a property on the class? vs just being local to this constructor

Reply:
yeah it does because I clear this interval in the destructor of the component. if it was in module scope then the timer would run forever, even after the component is destroyed.